app/src/utils/data/ipfs.ts

```diff
@@ -54,7 +55,7 @@ export const getMetaPtr = ({ cid }: { cid: string | undefined }) => {
  */
 export const formatMetaPtr = (cid: string): MetaPtr => {
   return {
-    protocol: 1, // IPFS has a protocol ID of 1
+    protocol: cid ? 1 : 0, // IPFS has a protocol ID of 1
```

Comment:
Why is this needed? We shouldn't have to special case protocol 0 within the `ipfs` (protocol 1) file

Reply:
isn't this where we want to make the adjustment to the object?

Reply:
I think that would be better handled in the utils method that wraps around this method, similarly to help keep the app vs. protocol separation clean

Reply:
Can you point me to the file / line? This is what's being returned to be uploaded to ipfs, right? I assumed that's where this issue was https://github.com/dcgtc/dgrants/pull/522#discussion_r753219386

Reply:
I’m on mobile right now about to take off on a flight, but I think I see what’s happening 

You’re using this in the upload logo step, which is coupled to IPFS, which is why you put it here. I thought this was for parsing the logo

It still feels a little dirty IMO to have it here instead of a createEmptyPtr() method that those files fallback to if no image is uploaded

Reply:
ah I see, I take a look at the method - but is there a different place we want to adjust the object? Isn't this where we want to make the change to the object if no logo is uploaded?

Reply:
If we’re setting the logo pointer of metadata JSON to be empty and point to nothing, that is entirely independent of IPFS, which is why I don’t think it should be handled in this file. You can do that in utils or just generate it directly in the file(s?) that needs it

Reply:
I do think there is a way around modifying the object - is that what you're suggesting? i.e your option 2 https://github.com/dcgtc/dgrants/pull/522#discussion_r753219386